### PR TITLE
saidcetinkaya JS2 Week3 Homework

### PR DESCRIPTION
**Pull request by saidcetinkaya**

### CODE REVIEW

File: Week3/homework/step2-2.js [hunk 1]
```diff
@@ -3,18 +3,25 @@
 function threeFive(startIndex, stopIndex, threeCallback, fiveCallback) {
   const numbers = [];
 
-  // Replace this comment and the next line with your code
-  console.log(startIndex, stopIndex, threeCallback, fiveCallback, numbers);
+  for (let number = startIndex; number < stopIndex + 1; number++) {
+    numbers[number - startIndex] = number;
```
Comment (paulbremer) [inline Week3/homework/step2-2.js]: What does this line do?

Comment (saidcetinkaya) [reply Week3/homework/step2-2.js]: It’s creating index for array and push a value to respective index.

Comment (paulbremer) [reply Week3/homework/step2-2.js]: But you are not using that array (`numbers`) anymore so no need to push the value there.

Comment (saidcetinkaya) [reply Week3/homework/step2-2.js]: yes, you're right. I fixed it now. Thank you! :)

Comment (paulbremer) [reply Week3/homework/step2-2.js]: After some more reviewing I now understand that you were doing the following: 'The function should first generate an array containing values from start value to end value (inclusive).'

My mistake, sorry, the `numbers` does contain the correct numbers for the array! 🙈

Comment (saidcetinkaya) [reply Week3/homework/step2-2.js]: still, it was unnecessary 🙈

Comment (saidcetinkaya) [reply Week3/homework/step2-2.js]: FYI: and also I changed the 2.7

**Pull request closed without merging**